src/Microsoft.DotNet.Darc/src/DarcLib/Models/Darc/DependencyGraph.cs

```diff
@@ -456,6 +456,13 @@ public class DependencyGraph
                 }
                 return dependencies;
             }
+            catch (DependencyFileNotFoundException exc)
```

Comment:
Also, on a second review, looks like exc is not used anywhere, you could just do `catch (DependencyFileNotFoundException exc)`